### DISCUSSION TITLE
Module 'Brownie' no where to be found

### DISCUSSION THREAD

**Discussion by shucks18** [smartcontractkit/full-blockchain-solidity-course-py#945] [Q&A] [answered]
Im around 4:35:00 in the video where patrick imports accounts from brownie, but I can't seem to find the module anywhere (and yes :( I have installed brownie )

**Comment by freddie71010** [comment]
It's possible you were working inside a virtualenv when you installed `brownie` and now have exited it. If you have installed brownie and then type `brownie --version` in your terminal, you should get a version number back.

**Reply by shucks18** [reply]
yep i do get a version back but there's still no module

**Reply by shucks18** [reply]
appreciate the help tho :)

**Comment by cromewar** [comment, marked as answer]
Hello @shucks18 
The most likely scenarios are:

1. You are not choosing the correct python interpreter on vscode (click on the bottom left corner and select the one you have installed brownie).
2. You have installed brownie using pipx, which causes problems with imports on vscode, so I strongly suggest you to use regular pip instead.
I hope this info might help.

**Reply by shucks18** [reply]
so it was actually (2) I installed brownie using pipx. installed using pip and it worked... once again @cromewar appreciate all the help!

**Reply by cromewar** [reply]
I'm glad is solved now ;)

**Reply by Python-David** [reply]
You are a life saver! @cromewar

**Reply by cromewar** [reply]
thanks @Python-David !!
